# CriteoApiError

Criteo API response error

## Properties
Name | Type | Description | Notes
------------ | ------------- | ------------- | -------------
**trace_id** | **str, none_type** | The correlation ID provided by the gateway | [optional] 
**type** | **str, none_type** | A machine-readable code specifying error category | [optional] 
**code** | **str, none_type** | A machine-readable error code string in kabab-case. Unique across Criteo | [optional] 
**instance** | **str, none_type** | A URI reference that identifies the specific occurrence of the problem | [optional] 
**title** | **str, none_type** | A short, human-readable remarks of the problem type. | [optional] 
**detail** | **str, none_type** | A human-readable explanation specific to this occurrence of the problem. | [optional] 
**source** | **{str: (str,)}, none_type** | A machine-readable structure to reference to the exact location(s) causing the error(s). | [optional] 
**any string name** | **bool, date, datetime, dict, float, int, list, str, none_type** | any string name can be used but the value must be the correct type | [optional]

[[Back to Model list]](../README.md#documentation-for-models) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to README]](../README.md)


